docs/docs/artifact-store.md

```diff
@@ -150,25 +143,29 @@ Server: Jetty(8.1.11.v20130520)
 ```
 
 
-## Automatic Artifact Storing
+## Automatic Artifact Storage
+
+An AppDefinition holds a sequence of URIs that Marathon fetches when it starts each app instance.
 
-An AppDefinition holds a sequence of URIs, that get fetched on each instance, that gets started.
 The artifact could be fetched directly from the source, or put into the artifact store.
 One simple way to do this is automatic artifact storing.
 
-The AppDefinition has a field storeUrls, which holds an array of URL strings.
+You can automatically store artifacts in the artifact store with the ```storeURLs``` field of the AppDefinition. The ```storeURLs``` field holds an array of URL strings.
 Every URL here is processed in this way:
 
-* The URL gets downloaded
-* The byte stream is stored in the asset store
-* The asset store url is added to the AppDefinition uris list
-* The url is removed from the storeUrls array
+* The URL is downloaded. 
+* The byte stream is stored in the asset store.
+* Marathon adds the asset store URL to the AppDefinition URIs list.
```

Comment:
Yes: the original url (where this asset is downloaded from) is replaced by the store url.